### PR TITLE
Changes to the import samples script with new naming conventions

### PR DESCRIPTION
**Pull request by withrocks**

### CODE REVIEW

File: clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py [hunk 1]
```diff
@@ -4,102 +4,127 @@
...
-    def execute(self):
-        self.control_mapping = dict()
-        for mapping in self.context.current_step.udf_control_mapping.split(","):
-            map_from, map_to = mapping.split(":")
-            self.control_mapping[map_to] = re.compile(map_from)
+    Requires a CSV file with the headers barcode;well 
+
+
+    Creates two containers with samples and controls in Clarity:
+        COVID_PREXT_<timestamp>
```
Comment (GitEdvard) [inline clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py:28]: The naming scripts for the RNA and PCR plates doesn't work together with this naming convention

File: clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py [hunk 2]
```diff
@@ -4,102 +4,127 @@
...
+    def create_sample(self, original_name, timestamp, project, specifier):
+        name = map(str, [original_name, timestamp])
+        if specifier:
+            name.append(specifier)
+        name = "_".join(name)
+        sample = Sample(sample_id=None, name=name, project=project)
+        sample.udf_map.force("Control", "No")
+        return sample
+
+    def create_control(self, original_name, control_type, timestamp, running, project, specifier):
```
Comment (GitEdvard) [inline clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py:49]: running_number?

File: clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py (hunk 1 above)
Comment (withrocks) [reply clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py:28]: Yeah, but this is what was decided on the meeting yesterday if I'm not mistaken. I guess we'll have to update those then, but I'd prefer to do it in another PR.

File: clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py (hunk 2 above)
Comment (withrocks) [reply clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py]: Fixed!

**Pull request merged**